content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Multiprotocol Shares"
+description: "Provides instructions on setting up SMB and NFSv4 mixed-mode shares."
+weight: 40
+tags:
+- scaleshares
+- scalesmb
+- scalenfs
+---
+
+{{< toc >}}
+
+## About Multiprotocol Shares
+
+A multiprotocol or mixed-mode NFS and SMB share supports both NFS and SMB protocols for sharing data.
+Multiprotocol shares allow clients running different operating systems to access the same data.
+This can be particularly useful in environments with a mix of Unix-like systems (which prefer NFS) and Windows systems (which prefer SMB).
```

Comment:
We need to emphasize that the preferred configuration is to use the Linux SMB client rather than a multiprotocol share.